steps/apptestai-test/0.1.0/step.yml

```diff
@@ -0,0 +1,91 @@
+title: apptest.ai Test
+summary: |
+  bitrise step for apptest.ai test execution
```

Comment:
These strings are user-facing, could you please fix punctuation/capitalization for all titles, summary and description? Example: Bitrise instead of bitrise, titles should consistently start capitalized, punctuation should be consistent.

Reply:
Thank you for the review. Modified and pushed a new commit.